src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/AlertCodesController.kt

```diff
@@ -107,6 +109,75 @@ class AlertCodesController(
     httpRequest: HttpServletRequest,
   ) = alertCodeService.deactivateAlertCode(alertCode, httpRequest.alertRequestContext())
 
+  @PreAuthorize("hasAnyRole('$ROLE_ALERTS_ADMIN')")
```

Comment:
I think this is too restrictive? Getting alert types and codes should be available to all so maybe `@PreAuthorize("hasAnyRole('$ROLE_ALERTS_READER', '$ROLE_ALERTS_ADMIN', '$PRISON')")`? The latter is a role all DPS users have

Reply:
Done.